app/views/dashboards/_search_form.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= t 'dashboard.search.header' %>
   </h1>
 
-  <%= bootstrap_form_with url: '/search', layout: :inline do |p| %>
+  <%= bootstrap_form_with url: '/search', layout: :inline, local: false do |p| %>
```

Comment:
is this one a change, or just an implicit made explicit?

Reply:
yep, this is an implicit made explicit!